proto/ibc-go-vendor/cosmos/base/tendermint/v1beta1/query.proto

```diff
@@ -0,0 +1,208 @@
+syntax = "proto3";
+package cosmos.base.tendermint.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "google/protobuf/any.proto";
+import "google/api/annotations.proto";
+import "tendermint/p2p/types.proto";
+import "tendermint/types/types.proto";
+import "cosmos/base/query/v1beta1/pagination.proto";
+import "cosmos/base/tendermint/v1beta1/types.proto";
+import "cosmos_proto/cosmos.proto";
+import "tendermint/types/block.proto";
+
+option go_package = "github.com/cosmos/cosmos-sdk/client/grpc/tmservice";
+
+// Service defines the gRPC querier service for tendermint queries.
+// TODO(erwan): rename this back to Service, investigate the ramifications of not doing so
+service TendermintQuery {
+  // GetNodeInfo queries the current node info.
+  rpc GetNodeInfo(GetNodeInfoRequest) returns (GetNodeInfoResponse) {
+    option (google.api.http).get = "/cosmos/base/tendermint/v1beta1/node_info";
```

Comment:
No idea what the right answer is here, but should we be annotating these with `cosmos` paths, if we're not doing a cosmos-sdk thing, and thinking of this proto file as something we want to upstream? Or should we just call them `tendermint/blah`?

Reply:
Agreed! `tendermint/query/node_info` would work. I'm not sure why the Cosmos SDK has different [names](https://docs.cosmos.network/v0.44/core/proto-docs.html#cosmos/base/tendermint/v1beta1/query.proto) for what I think (?) resolves to a [`Status`](https://docs.tendermint.com/master/spec/rpc/#status).

It'd be tempting to just stick to the spec described in the [Tendermint RPC](https://docs.tendermint.com/master/spec/). I'm still getting up-to-speed about the ecosystem, so I'll defer to you on this one